### DISCUSSION TITLE
How to publish a project to local IIS

### DISCUSSION THREAD

**Discussion by hehetiger** [OrchardCMS/OrchardCore#16638] [Q&A]
The interface for a normal web project, right-click on the MVC project and click publish, looks like this
![企业微信截图_20240830114148](https://github.com/user-attachments/assets/bc869f49-cd2b-4117-8800-6fe100c7af63)
![企业微信截图_20240830114209](https://github.com/user-attachments/assets/b6b527ed-33e2-4482-8ebc-93e1678ad547)
But the click to publish for the Orchard project is like this
![企业微信截图_20240830114227](https://github.com/user-attachments/assets/3a14700f-d8a1-4719-a599-9f3a2a188a3c)
I have never seen such a publishing interface before. Is there a detailed tutorial on how to publish downloaded Orchard Core projects to IIS

**Comment by hishamco** [comment]
First of all hope to share an English interface next time because we will lost :) regarding Orchard Core it's built on top of ASP.NET Core, so you can follow any tutorial or guideline for publishing an ASP.NET Core app to IIS

https://learn.microsoft.com/en-us/aspnet/core/tutorials/publish-to-iis?view=aspnetcore-8.0&tabs=visual-studio
https://dotnettutorials.net/lesson/how-to-host-asp-net-core-web-app-into-local-iis-server/

**Comment by weirdyang** [comment]
It's the publish profile used. Just a create a new one, and choose publish to folder
<img width="466" height="408" alt="image" src="https://github.com/user-attachments/assets/c00efc00-af12-495e-ae29-b62eb403c954" />
